xpk.py

```diff
@@ -229,6 +229,14 @@
         command: [ "sleep", "inf" ]
 """
 
+cluster_configmap_yaml = """kind: ConfigMap
+apiVersion: v1 
+metadata:
+  name: {args.cluster}-resources-configmap 
```

Comment:
Not needed now but when we add the cluster version in a new config map, we can change this line to
`name: {args.cluster}-{config_map_name}-configmap`, then we can reuse this config map. Up to you if you want to add that now or later!